appeals/api/src/server/repositories/appeal-lists.repository.js

```diff
@@ -19,9 +19,8 @@ import { getEnabledAppealTypes } from '#utils/feature-flags-appeal-types.js';
  * @param {number} inspectorId
  * @param {number} caseOfficerId
  * @param {boolean} isGreenBelt
- * @returns {Promise<[number, Omit<Appeal, 'parentAppeals' | 'childAppeals'>[]]>}
```

Comment:
I've exported the inferred return types of these funcs so the types are generated by Prisma